source/recurring_payments/index.html.md.erb

```diff
@@ -37,17 +37,17 @@ You can [read more about the lifecycle of agreements](/recurring_payments/#under
 
 In this integration example, the service creates an agreement when a user says they want to make recurring payments, then uses that agreement to continue taking payments.
 
-![A sequence diagram showing the journey of creating, setting up, and using an agreement to take recurring payments.](/images/recurring-payments-success-sequence.svg)
+![](/images/recurring-payments-success-sequence.svg)
```

Comment:
Is this change intentional? if so, any reason to not have Alt text here?

Reply:
This change is intentional. Because all of the information from the diagram is also in the body text, the diagram doesn't need alt text. If we include alt text, then we're duplicating information, which is bad for screen reader users.

I forgot why we didn't need it myself so I had to double check with the content community but you can see their answers here - https://gds.slack.com/archives/CACV3TACU/p1670937867761379